contrib/endpoints/src/api_manager/config_test.cc

```diff
@@ -304,6 +304,7 @@ static const char auth_config[] =
     "    id: \"provider-id1\"\n"
     "    issuer: \"issuer1@gserviceaccount.com\"\n"
     "    jwks_uri: \"https://www.googleapis.com/jwks_uri1\"\n"
+    "    audiences: \"ok_audience1\"\n"
```

Comment:
It is already there. There are two apis with AuthRequirements and both have audiences right now. I have just moved one out to AuthProvider and the other one has AuthRequirements audience set.